modules/renderer/background_source.js

```diff
@@ -452,7 +452,11 @@ rendererBackgroundSource.Esri = function(data) {
 
 
     esri.getMetadata = function(center, tileCoord, callback) {
-        if (esri.id !== 'EsriWorldImagery') {
+        let mapServerUrl = esri.metadata;
+        if (esri.id === 'EsriWorldImagery') {
+            mapServerUrl = 'https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer';
```

Comment:
It’s pretty annoying that iD hard-codes the metadata MapServer URL for just Esri World Imagery, as if it’s the only ArcGIS ImageServer that comes with such an endpoint (far from it). osmlab/editor-layer-index#909 tracks specifying these endpoints upstream in the original data.

It would be a little tricky to generalize this code to work for every ArcGIS ImageServer. Layer number 4 also applies to the Wayback ImageServers and [wayback](https://github.com/vannizhang/wayback/) application depends on that, but I’ve encountered third-party ImageServers that put the metadata in other layers, for example Ohio’s OSIP ImageServer.